morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -156475,6 +156574,8 @@ Phthi_o_ta
 Phthi_-o_tes		
 :le:Phthiotes
 :no:Phthi_-o_t	es_ae masc
+#Ovid
```

Comment:
I'm using the `#@` sequence to denote special comments that we want the analyzer code to pick up on later - if that was the intention, please change this to `#@Ovid` (otherwise if it's just an informational comment then no need to do anything)

Reply:
Just informational - it's clear that these are the same word but I didn't see that variant spelling in any of the dictionaries so I wanted to note where it was coming from